code/game/objects/items/storage/book.dm

```diff
@@ -229,14 +229,14 @@ GLOBAL_LIST_INIT(bibleitemstates, list("bible", "koran", "scrapbook", "burning",
 			qdel(sword)
 	else if(istype(A, /obj/item/soulstone) && !iscultist(user))
 		var/obj/item/soulstone/SS = A
-		if(SS.purified)
+		if(SS.theme == "holy")
```

Comment:
Okay, so *here* you're using the values of the defines but without the define.

Move the defines out to something in `__DEFINES`, and modify this to use that define.

Reply:
can do